README.md

```diff
@@ -8,16 +8,51 @@ A composition library, which works well in a TypeScript environment.
 
 **WARNING** This is *beta* software.  While we do not anticipate significant changes to the API at this stage, we may feel the need to do so.  This is not yet production ready, so you should use at your own risk.
 
-This library embraces the concepts of "composition" versus classical Object Oriented inheritance.  The classical model follows a pattern whereby you add functionality to an ancestor by extending it.  Subsequently all other descendants from that class will also inherit that functionality.
+## Background
+
+In creating this library, we were looking to solve the following problems with Classes and inheritance in ES6+ and TypeScript:
+
+* Single inheritance: ES6 classes are essentially syntactic sugar around prototypical inheritance. A descent class can only derive from a single ancestor class
```

Comment:
I think this one is valid British English, but not usual for Americans to hear. I'll change it. :)